README.md

```diff
@@ -1 +1,40 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Webpage
+
+> My personal portfolio webpage (under development).
+
+![screenshot](./screenshot.png)
+
+Webpage I use to show my work and selling my services.
+
```

Comment:
- [ ] [OPTIONAL] For this and the next of your projects, it would be great if you add more descriptive presentation for your repositories, just try to explain what you did, what kind of technologies or techniques you used, what is the mean scope, and so on.